lib/ReactViews/Custom/Chart/MomentPointsChart.jsx

```diff
@@ -62,26 +63,42 @@ class MomentPointsChart extends React.Component {
     const { id, chartItem, scales } = this.props;
     const baseKey = `moment-point-${chartItem.categoryName}-${chartItem.name}`;
     const fillColor = chartItem.getColor();
+    const isClickable = chartItem.onClick !== undefined;
+    const clickProps = point => {
+      if (isClickable) {
+        return {
+          pointerEvents: "all",
+          cursor: "pointer",
+          onClick: () => chartItem.onClick(point)
+        };
+      }
+      return {};
+    };
     return (
       <g id={id}>
         <For each="p" index="i" of={this.points}>
-          <Circle
+          <StyledCircle
             key={`${baseKey}-${i}`}
             cx={scales.x(p.x)}
             cy={scales.y(p.y)}
             r={3}
             fill={fillColor}
             opacity={p.isSelected ? 1.0 : 0.3}
-            pointerEvents="all"
-            cursor="pointer"
-            onClick={() => chartItem.onClick(p)}
+            {...clickProps(p)}
           />
         </For>
       </g>
     );
   }
 }
 
+const StyledCircle = styled(Circle)`
+  &:hover {
+    opacity: 1;
+    fill: white;
+  }
+`;
+
```

Comment:
Adds a hover effect on the point (Circle).

Reply:
up to you but do we need this hover effect if the item is not clickable?

Reply:
Yes, I think it makes it easier for the user to identify the point on which the mouse is hovering.